### PR TITLE
Recitfy docs links, synchronize READMEs

### PR DESCRIPTION
**Pull request by akarve**

### CODE REVIEW

File: catalog/app/containers/Bucket/Dir.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ import * as NamedRoutes from 'utils/NamedRoutes'
 import Link from 'utils/StyledLink'
 import { getBreadCrumbs, ensureNoSlash, withoutPrefix, up } from 'utils/s3paths'
 
+import docs from 'constants/urls'
```
Comment (nl0) [inline catalog/app/containers/Bucket/Dir.js]:
```suggestion
import { docs } from 'constants/urls'
```

File: catalog/app/containers/Bucket/PackageList.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ import * as NamedRoutes from 'utils/NamedRoutes'
 import Link from 'utils/StyledLink'
 import { readableQuantity } from 'utils/string'
 
+import docs from 'constants/urls'
```
Comment (nl0) [inline catalog/app/containers/Bucket/PackageList.js]:
```suggestion
import { docs } from 'constants/urls'
```

File: catalog/app/containers/Bucket/errors.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { createStructuredSelector } from 'reselect'
 import Button from '@material-ui/core/Button'
 
 import * as Auth from 'containers/Auth'
+import docs from 'constants/urls'
```
Comment (nl0) [inline catalog/app/containers/Bucket/errors.js]:
```suggestion
import { docs } from 'constants/urls'
```

File: catalog/package-lock.json [hunk 4]
```diff
@@ -5888,8 +5888,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```
Comment (nl0) [inline catalog/package-lock.json]: it's strange that npm doing this again. which version are you using?

Comment (akarve) [reply catalog/package-lock.json]: I'm on 6.4.1. It never stopped doing this weirdness.

**Pull request merged**